common/meta/raft-store/build.rs

```diff
@@ -17,16 +17,15 @@
 use std::env;
 use std::path::Path;
 fn main() {
-    common_building::setup();
```

Comment:
Isn't the proto gonna moved into a standalone crate? 

The proto is about RPC protocol. IMHO, it has nothing to do with raft-store. Unless I missed anything. 🤔

Reply:
Cause I encounter the rust orphan rule :(

Reply:
> The proto is about RPC protocol. IMHO, it has nothing to do with raft-store. Unless I missed anything. 🤔

+1

how about calling it meta-proto or meta-api? q.q